### ISSUE TITLE
"Attribute Name" dropdown in item field is ambiguous

### ISSUE THREAD

**Issue by objecttothis** [opensourcepos/opensourcepos#2222]
### Background information

__IMPORTANT: If you choose to ignore this issue report template, your issue will be closed as we cannot help without the requested information.__

Please make sure you tick (add an x between the square brackets with no spaces) the following check boxes:

- [] Reporting an issue of an unmodified OSPOS installation
- [] Checked [open and closed issues](https://github.com/opensourcepos/opensourcepos/issues?utf8=%E2%9C%93&q=is%3Aissue) and no similar issue was already reported (please make sure you searched!)
- [] Read [README](https://github.com/opensourcepos/opensourcepos/blob/master/README.md), [WHATS_NEW](https://github.com/opensourcepos/opensourcepos/blob/master/WHATS_NEW.txt), [INSTALL.md](https://github.com/opensourcepos/opensourcepos/blob/master/INSTALL.md) and [UPGRADE](https://github.com/opensourcepos/opensourcepos/blob/master/UPGRADE.txt)
- [] Read the [FAQ](https://github.com/opensourcepos/opensourcepos#faq) for any known install and/or upgrade gotchas (in specific PHP extensions installed)
- [] Read the [wiki](https://github.com/opensourcepos/opensourcepos/wiki)
- [] Executed any database upgrade scripts if an upgrade pre 3.0.0 (e.g. database/2.4_to_3.0.sql)
- [] Aware the installation code is in [bintray](https://bintray.com/jekkos/opensourcepos/opensourcepos/view/files?sort=updated&order=asc#files) (see README), and [GitHub master](https://github.com/opensourcepos/opensourcepos/tree/master) is for [developers only](https://github.com/opensourcepos/opensourcepos/wiki/Development-setup) and therefore not complete nor stable

### Installation information

- OSPOS version is: 3.3.0
- OSPOS git commit hash is: 465a2c
- PHP version is: 5.6
- MySQL or MariaDB version is: MySQL 5.7.2
- OS and version is: FreeBSD 11.0
- WebServer is: Apache 2.4
- Selected language is: English
- (If applicable) Docker installation: 
- (If applicable) Installation package for the LAMP/LEMP stack is: 

### New Feature

When editing an item the dropdown field allowing you to assign an attribute to the item is currently titled "Attribute Name" but this is ambiguous as it doesn't give clarity as to what the dropdown does.  I think this would be better worded if it were titled "Add Attribute" and "Nothing Selected." was changed to "[Select Attribute]" or some other variation.  Further I think it would be better if this appeared below the list of attributes.  This is because once the initial attributes are assigned to an item this dropdown will rarely ever be used, so if it appears below the attributes it allows the user to see the most pertinent information available.

As a bonus (but separate) thought, I think that we should make the "Attribute Name" (or whatever we end up calling it) invisible when there are no attributes to add.  This makes for a cleaner interface.

**Comment by objecttothis** [comment]
The wording changes I can make pretty easily, but all that is dependent on #2217

**Comment by jekkos** [comment]
@objecttothis Any idea on how we should rename this label then ?

**Comment by objecttothis** [comment]
@jekkos I made the changes to the language in English to add clarity.  What do you guys think about moving the "Add Attribute" drop-down to below the existing attributes so that the most valuable information is visible first?

**Comment by objecttothis** [comment]
This is why I am implementing #1909 so that default persistent columns can be selected which will prevent large amount of scrolling needed unless the admin selects a large number of default columns to display or the user selects a large number of columns to display which are then stored in their session data.

**Comment by jekkos** [comment]
@objecttothis I would agree adding it only if indeed we have this feature to 'add attributes by default'. I think for the implementation we might just stick to using the flag on the definition to show it by default, so no entries in the link table whatsoever.

Or did you have something in mind  already?

**Comment by objecttothis** [comment]
This might clean up the New Item view a little. My only thought is that if New Item and Edit Item will use the same view then it makes the Attribute data a little less accessible.

**Comment by erastus** [comment]
@objecttothis  New and edition. Share the view, no problem.

**Comment by erastus** [comment]
@objecttothis  a tab for the attributes would be good, it's more accessible, right now it looks very crowded. I only suggest. Regards!

**Comment by objecttothis** [comment]
@erastus I'm going to close this issue since the issue itself has been completed and merged into the master.  Please create a new issue regarding the tab enhancement request.  This needs to be done to keep everything clean and simple.
